lib/core/form/services/widget-visibility.service.ts

```diff
@@ -91,6 +94,19 @@ export class WidgetVisibilityService {
 
     }
 
+    private isValidFieldValue(form: FormModel, visibilityObj: WidgetVisibilityModel): boolean {
+        let isValid = true;
+        if (visibilityObj.leftType === WidgetTypeEnum.field) {
+            form.getFormFields().forEach((formField: FormFieldModel) => {
+                if (this.isSearchedField(formField, visibilityObj.leftFormFieldId)) {
```

Comment:
It's not a good practice to interrupt a forEach this way. I guess that you're trying to find the field among all the fields. 
Can we use ```form.getFormFields().find((formField: FormFieldModel) => your stuff)``` instead?

Reply:
Updated.